api/envoy/config/filter/http/grpc_stats/v2alpha/config.proto

```diff
@@ -0,0 +1,19 @@
+syntax = "proto3";
```

Comment:
This isn't wired into docs generation (I noticed as it's not doing protoxform generation of v3).